lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -1229,6 +1232,12 @@ instance Arbitrary a => Arbitrary (AddressAmount a) where
     shrink _ = []
 
 instance Arbitrary (PostTransactionData t) where
+    arbitrary = PostTransactionData
+        <$> arbitrary
+        <*> arbitrary
+        <*> elements [Just SelfWithdrawal, Nothing]
```

Comment:
we don't want to include external withdrawals here?

Reply:
This is a bit tricky because the swagger2 Haskell libraries doesn't support the `oneOf` constructor :( ... And here, the swagger schema is a oneOf between a string and a mnemonic sentence; so I had to pick one to not overly complexifies things here; since the mnemonic sentences schema and JSON are already well-tested through other API types, I opted for the self withdrawal one which is completely new and only exists in this context.